src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
I found out to add this to add allowing mocks of final classes. Should help with mocking in the future.

Reply:
Interesting, it looks like it is just included in mockito2 it might be worth filing a ticket to just upgrading that.

Reply:
I think we are using mockito2? At least given by the version number in the pom.

Reply:
Adding this is straight from Mockito's docs: https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2#unmockable